middleware/proxy_test.go

```diff
@@ -302,3 +305,73 @@ func TestProxyRewriteRegex(t *testing.T) {
 		})
 	}
 }
+
+func TestProxy_1_11(t *testing.T) {
```

Comment:
Is this related to Go 1.11 somehow or why is its name suffixed with `_1_11`? (Just curios, looks like a normal proxy test)